backend/src/main/java/it/chalmers/gamma/service/FKITGroupService.java

```diff
@@ -100,7 +105,7 @@ public void editGroupAvatar(FKITGroupDTO groupDTO, String url) throws GroupDoesN
         this.repo.save(group);
     }
 
-    protected FKITGroup getGroup(FKITGroupDTO group) {
+    public FKITGroup getGroup(FKITGroupDTO group) {
```

Comment:
This should not be public, not having the DTOs be accessible from outside the service layer is good practise